homework/01LessonOne.js

```diff
@@ -202,13 +203,15 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // What’s the difference between == and ===?
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: '==' is a comparison operator and '===' is a strict
+// comparison operator. The == operator will compare the values of strings to
+// numbers and the strict comparison operator will not.
 //
 // What’s the difference between != and !==?
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: "!=" is a not equals comparison operator and
+// '!==' is a strict not equals comparison operator that compares the falsiness
+// similar to the way === compares the strict truthiness.
 // ****
 
```

Comment:
Awesome!! All them tests be passin'

![fin](http://media2.giphy.com/media/rOTGSPxvJJY7m/giphy.gif)